Automattic/jetpack
# Stats Page: Add VideoPress nudge

#### Changes proposed in this Pull Request:

* This PR adds one more nudge to the stats page. This card only appears if the user doesn't have Jetpack Complete or Jetpack VideoPress.

#### Jetpack product discussion


Context: pbNhbs-18H-p2

#### Does this pull request change what data or activity we track or use?


Not directly.

#### Testing instructions:






![image](https://user-images.githubusercontent.com/5714212/138509627-cce7c038-ae9a-4beb-8eca-242fb4c18406.png)

* Go to the detailed stats page: `/wp-admin/admin.php?page=stats`
* If you have Jetpack Complete or Jetpack VideoPress, it shouldn't show an extra nudge item
* If you don't have any of the products above, make sure a nudge item appears and it looks like the screenshot above
* Confirm the Upgrade button directs you to `/wp-admin/admin.php?page=jetpack#/product/videopress` (the page won't work if #21497 is not merged yet)
